src/AddTodoForm.tsx

```diff
@@ -0,0 +1,22 @@
+const AddTodoForm = () => (
+    <div id="addTodoForm-wrapper">
+        <form id="addTodoForm">
+            <label htmlFor="todoTitle">
+                Title
+            </label>
+            <input id="todoTitle"></input>
+        </form>
+{/* @Ethan [HTML SEMANTIC/QUESTION/DISCUSSION] 
+input type="submit"
+button type="submit"
+parent form, div
+
+type="submit" is the default with button
+Book says "Next, create a submit <button> element with text "Add" in the assignment"
+*/}
+        <button type="submit" form="addTodoForm" value="Add">Add</button>
```

Comment:
You can abandon the form property inside your `button` if you place the button inside the form. 

You can remove the type="submit" since it is the default option. Sometimes the assignment wording isn't the best.

Reply:
gotcha thanks.